packages/upload-api/src/lib.js

```diff
@@ -184,7 +184,7 @@ export const createService = (context) => ({
     ...createAdminService(context),
     // @ts-expect-error `uploadTable` items now have a `cause` field. This does
     // not matter since `admin/store/inspect` handler does not use this table.
-    store: createLegacyAdminService(context).store
+    store: createLegacyAdminService(context).store,
```

Comment:
??? I thought we were set to no trailing commas?

Reply:
No, [we use `"trailingComma": "es5"`](https://github.com/storacha/upload-service/blob/b7349ba0dea9fd3557bf1c0b1be79459ec7d0d1c/package.json#L34), which is trailing commas anywhere that ES5 won't complain about them (whereas modern ES is more permissive).